app/models/poll.rb

```diff
@@ -1,4 +1,7 @@
 class Poll < ActiveRecord::Base
+  acts_as_paranoid column: :hidden_at
+  include ActsAsParanoidAliases
```

Comment:
@zamuro this won't be neccesary, its not a requirement to make the Poll hidden

Reply:
When removing hidden_at from migration and these two lines:
```

  5) Polls Show user b can access to comments from user a
     Failure/Error: belongs_to :commentable, -> { with_hidden }, polymorphic: true, counter_cache: true
     
     ActionView::Template::Error:
       undefined local variable or method `with_hidden' for #<Poll::ActiveRecord_Relation:0x0055f8a558f4a0>
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/activerecord-4.2.9/lib/active_record/relation/delegation.rb:136:in `method_missing'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/activerecord-4.2.9/lib/active_record/relation/delegation.rb:99:in `method_missing'
     # ./app/models/comment.rb:22:in `block in <class:Comment>'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/activerecord-4.2.9/lib/active_record/associations/builder/association.rb:65:in `instance_exec'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/activerecord-4.2.9/lib/active_record/associations/builder/association.rb:65:in `block in initialize'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/activerecord-4.2.9/lib/active_record/associations/association_scope.rb:191:in `instance_exec'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/activerecord-4.2.9/lib/active_record/associations/association_scope.rb:191:in `eval_scope'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/activerecord-4.2.9/lib/active_record/associations/association_scope.rb:155:in `block (2 levels) in add_constraints'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/activerecord-4.2.9/lib/active_record/associations/association_scope.rb:154:in `each'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/activerecord-4.2.9/lib/active_record/associations/association_scope.rb:154:in `block in add_constraints'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/activerecord-4.2.9/lib/active_record/associations/association_scope.rb:141:in `each'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/activerecord-4.2.9/lib/active_record/associations/association_scope.rb:141:in `each_with_index'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/activerecord-4.2.9/lib/active_record/associations/association_scope.rb:141:in `add_constraints'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/activerecord-4.2.9/lib/active_record/associations/association_scope.rb:39:in `scope'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/activerecord-4.2.9/lib/active_record/associations/association_scope.rb:5:in `scope'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/activerecord-4.2.9/lib/active_record/associations/association.rb:97:in `association_scope'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/activerecord-4.2.9/lib/active_record/associations/association.rb:86:in `scope'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/activerecord-4.2.9/lib/active_record/associations/singular_association.rb:42:in `get_records'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/activerecord-4.2.9/lib/active_record/associations/singular_association.rb:57:in `find_target'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/activerecord-4.2.9/lib/active_record/associations/association.rb:138:in `load_target'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/activerecord-4.2.9/lib/active_record/associations/association.rb:53:in `reload'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/activerecord-4.2.9/lib/active_record/associations/singular_association.rb:9:in `reader'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/bullet-5.5.1/lib/bullet/active_record42.rb:215:in `reader'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/activerecord-4.2.9/lib/active_record/associations/builder/association.rb:115:in `commentable'
     # ./app/views/comments/_comment.html.erb:2:in `_app_views_comments__comment_html_erb___2900209252357399508_47263206653840'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/actionview-4.2.9/lib/action_view/template.rb:145:in `block in render'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/activesupport-4.2.9/lib/active_support/notifications.rb:166:in `instrument'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/actionview-4.2.9/lib/action_view/template.rb:333:in `instrument'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/actionview-4.2.9/lib/action_view/template.rb:143:in `render'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/actionview-4.2.9/lib/action_view/renderer/partial_renderer.rb:339:in `render_partial'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/actionview-4.2.9/lib/action_view/renderer/partial_renderer.rb:310:in `block in render'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/actionview-4.2.9/lib/action_view/renderer/abstract_renderer.rb:39:in `block in instrument'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/activesupport-4.2.9/lib/active_support/notifications.rb:164:in `block in instrument'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/activesupport-4.2.9/lib/active_support/notifications/instrumenter.rb:20:in `instrument'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/activesupport-4.2.9/lib/active_support/notifications.rb:164:in `instrument'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/actionview-4.2.9/lib/action_view/renderer/abstract_renderer.rb:39:in `instrument'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/actionview-4.2.9/lib/action_view/renderer/partial_renderer.rb:309:in `render'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/actionview-4.2.9/lib/action_view/renderer/renderer.rb:51:in `render_partial'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/actionview-4.2.9/lib/action_view/helpers/rendering_helper.rb:35:in `render'
     # ./app/views/shared/_comments.html.erb:25:in `block (2 levels) in _app_views_shared__comments_html_erb___3346234190162093915_47263181704260'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/activerecord-4.2.9/lib/active_record/relation/delegation.rb:46:in `each'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/activerecord-4.2.9/lib/active_record/relation/delegation.rb:46:in `each'
     # ./app/views/shared/_comments.html.erb:24:in `block in _app_views_shared__comments_html_erb___3346234190162093915_47263181704260'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/actionview-4.2.9/lib/action_view/helpers/cache_helper.rb:117:in `cache'
     # ./app/views/shared/_comments.html.erb:1:in `_app_views_shared__comments_html_erb___3346234190162093915_47263181704260'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/actionview-4.2.9/lib/action_view/template.rb:145:in `block in render'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/activesupport-4.2.9/lib/active_support/notifications.rb:166:in `instrument'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/actionview-4.2.9/lib/action_view/template.rb:333:in `instrument'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/actionview-4.2.9/lib/action_view/template.rb:143:in `render'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/actionview-4.2.9/lib/action_view/renderer/partial_renderer.rb:339:in `render_partial'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/actionview-4.2.9/lib/action_view/renderer/partial_renderer.rb:310:in `block in render'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/actionview-4.2.9/lib/action_view/renderer/abstract_renderer.rb:39:in `block in instrument'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/activesupport-4.2.9/lib/active_support/notifications.rb:164:in `block in instrument'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/activesupport-4.2.9/lib/active_support/notifications/instrumenter.rb:20:in `instrument'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/activesupport-4.2.9/lib/active_support/notifications.rb:164:in `instrument'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/actionview-4.2.9/lib/action_view/renderer/abstract_renderer.rb:39:in `instrument'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/actionview-4.2.9/lib/action_view/renderer/partial_renderer.rb:309:in `render'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/actionview-4.2.9/lib/action_view/renderer/renderer.rb:51:in `render_partial'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/actionview-4.2.9/lib/action_view/helpers/rendering_helper.rb:35:in `render'
     # ./app/views/polls/show.html.erb:46:in `_app_views_polls_show_html_erb__504409128911861219_47263182546540'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/actionview-4.2.9/lib/action_view/template.rb:145:in `block in render'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/activesupport-4.2.9/lib/active_support/notifications.rb:166:in `instrument'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/actionview-4.2.9/lib/action_view/template.rb:333:in `instrument'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/actionview-4.2.9/lib/action_view/template.rb:143:in `render'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/actionview-4.2.9/lib/action_view/renderer/template_renderer.rb:54:in `block (2 levels) in render_template'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/actionview-4.2.9/lib/action_view/renderer/abstract_renderer.rb:39:in `block in instrument'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/activesupport-4.2.9/lib/active_support/notifications.rb:164:in `block in instrument'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/activesupport-4.2.9/lib/active_support/notifications/instrumenter.rb:20:in `instrument'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/activesupport-4.2.9/lib/active_support/notifications.rb:164:in `instrument'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/actionview-4.2.9/lib/action_view/renderer/abstract_renderer.rb:39:in `instrument'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/actionview-4.2.9/lib/action_view/renderer/template_renderer.rb:53:in `block in render_template'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/actionview-4.2.9/lib/action_view/renderer/template_renderer.rb:61:in `render_with_layout'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/actionview-4.2.9/lib/action_view/renderer/template_renderer.rb:52:in `render_template'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/actionview-4.2.9/lib/action_view/renderer/template_renderer.rb:14:in `render'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/actionview-4.2.9/lib/action_view/renderer/renderer.rb:46:in `render_template'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/actionview-4.2.9/lib/action_view/renderer/renderer.rb:27:in `render'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/actionview-4.2.9/lib/action_view/rendering.rb:100:in `_render_template'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/actionpack-4.2.9/lib/action_controller/metal/streaming.rb:217:in `_render_template'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/actionview-4.2.9/lib/action_view/rendering.rb:83:in `render_to_body'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/actionpack-4.2.9/lib/action_controller/metal/rendering.rb:32:in `render_to_body'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/actionpack-4.2.9/lib/action_controller/metal/renderers.rb:37:in `render_to_body'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/actionpack-4.2.9/lib/abstract_controller/rendering.rb:25:in `render'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/actionpack-4.2.9/lib/action_controller/metal/rendering.rb:16:in `render'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/actionpack-4.2.9/lib/action_controller/metal/instrumentation.rb:44:in `block (2 levels) in render'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/activesupport-4.2.9/lib/active_support/core_ext/benchmark.rb:12:in `block in ms'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/activesupport-4.2.9/lib/active_support/core_ext/benchmark.rb:12:in `ms'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/actionpack-4.2.9/lib/action_controller/metal/instrumentation.rb:44:in `block in render'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/actionpack-4.2.9/lib/action_controller/metal/instrumentation.rb:87:in `cleanup_view_runtime'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/activerecord-4.2.9/lib/active_record/railties/controller_runtime.rb:25:in `cleanup_view_runtime'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/actionpack-4.2.9/lib/action_controller/metal/instrumentation.rb:43:in `render'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/actionpack-4.2.9/lib/action_controller/metal/implicit_render.rb:10:in `default_render'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/actionpack-4.2.9/lib/action_controller/metal/implicit_render.rb:5:in `send_action'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/actionpack-4.2.9/lib/abstract_controller/base.rb:198:in `process_action'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/actionpack-4.2.9/lib/action_controller/metal/rendering.rb:10:in `process_action'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/actionpack-4.2.9/lib/abstract_controller/callbacks.rb:20:in `block in process_action'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/activesupport-4.2.9/lib/active_support/callbacks.rb:117:in `call'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/activesupport-4.2.9/lib/active_support/callbacks.rb:555:in `block (2 levels) in compile'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/activesupport-4.2.9/lib/active_support/callbacks.rb:505:in `call'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/activesupport-4.2.9/lib/active_support/callbacks.rb:92:in `__run_callbacks__'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/activesupport-4.2.9/lib/active_support/callbacks.rb:778:in `_run_process_action_callbacks'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/activesupport-4.2.9/lib/active_support/callbacks.rb:81:in `run_callbacks'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/actionpack-4.2.9/lib/abstract_controller/callbacks.rb:19:in `process_action'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/actionpack-4.2.9/lib/action_controller/metal/rescue.rb:29:in `process_action'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/actionpack-4.2.9/lib/action_controller/metal/instrumentation.rb:32:in `block in process_action'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/activesupport-4.2.9/lib/active_support/notifications.rb:164:in `block in instrument'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/activesupport-4.2.9/lib/active_support/notifications/instrumenter.rb:20:in `instrument'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/activesupport-4.2.9/lib/active_support/notifications.rb:164:in `instrument'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/actionpack-4.2.9/lib/action_controller/metal/instrumentation.rb:30:in `process_action'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/actionpack-4.2.9/lib/action_controller/metal/params_wrapper.rb:250:in `process_action'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/activerecord-4.2.9/lib/active_record/railties/controller_runtime.rb:18:in `process_action'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/actionpack-4.2.9/lib/abstract_controller/base.rb:137:in `process'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/actionview-4.2.9/lib/action_view/rendering.rb:30:in `process'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/actionpack-4.2.9/lib/action_controller/metal.rb:196:in `dispatch'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/actionpack-4.2.9/lib/action_controller/metal/rack_delegation.rb:13:in `dispatch'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/actionpack-4.2.9/lib/action_controller/metal.rb:237:in `block in action'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/actionpack-4.2.9/lib/action_dispatch/routing/route_set.rb:74:in `dispatch'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/actionpack-4.2.9/lib/action_dispatch/routing/route_set.rb:43:in `serve'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/actionpack-4.2.9/lib/action_dispatch/journey/router.rb:43:in `block in serve'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/actionpack-4.2.9/lib/action_dispatch/journey/router.rb:30:in `each'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/actionpack-4.2.9/lib/action_dispatch/journey/router.rb:30:in `serve'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/actionpack-4.2.9/lib/action_dispatch/routing/route_set.rb:817:in `call'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/turnout-2.4.0/lib/rack/turnout.rb:25:in `call'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/omniauth-1.6.1/lib/omniauth/strategy.rb:407:in `call_app!'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/omniauth-1.6.1/lib/omniauth/strategy.rb:265:in `mock_call!'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/omniauth-1.6.1/lib/omniauth/strategy.rb:184:in `call!'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/omniauth-1.6.1/lib/omniauth/strategy.rb:167:in `call'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/omniauth-1.6.1/lib/omniauth/strategy.rb:407:in `call_app!'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/omniauth-1.6.1/lib/omniauth/strategy.rb:265:in `mock_call!'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/omniauth-1.6.1/lib/omniauth/strategy.rb:184:in `call!'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/omniauth-1.6.1/lib/omniauth/strategy.rb:167:in `call'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/omniauth-1.6.1/lib/omniauth/strategy.rb:407:in `call_app!'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/omniauth-1.6.1/lib/omniauth/strategy.rb:265:in `mock_call!'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/omniauth-1.6.1/lib/omniauth/strategy.rb:184:in `call!'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/omniauth-1.6.1/lib/omniauth/strategy.rb:167:in `call'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/rack-attack-5.0.1/lib/rack/attack.rb:147:in `call'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/bullet-5.5.1/lib/bullet/rack.rb:12:in `call'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/warden-1.2.7/lib/warden/manager.rb:36:in `block in call'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/warden-1.2.7/lib/warden/manager.rb:35:in `catch'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/warden-1.2.7/lib/warden/manager.rb:35:in `call'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/rack-1.6.8/lib/rack/etag.rb:24:in `call'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/rack-1.6.8/lib/rack/conditionalget.rb:25:in `call'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/rack-1.6.8/lib/rack/head.rb:13:in `call'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/actionpack-4.2.9/lib/action_dispatch/middleware/params_parser.rb:27:in `call'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/actionpack-4.2.9/lib/action_dispatch/middleware/flash.rb:260:in `call'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/rack-1.6.8/lib/rack/session/abstract/id.rb:225:in `context'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/rack-1.6.8/lib/rack/session/abstract/id.rb:220:in `call'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/actionpack-4.2.9/lib/action_dispatch/middleware/cookies.rb:560:in `call'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/activerecord-4.2.9/lib/active_record/query_cache.rb:36:in `call'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/activerecord-4.2.9/lib/active_record/connection_adapters/abstract/connection_pool.rb:653:in `call'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/actionpack-4.2.9/lib/action_dispatch/middleware/callbacks.rb:29:in `block in call'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/activesupport-4.2.9/lib/active_support/callbacks.rb:88:in `__run_callbacks__'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/activesupport-4.2.9/lib/active_support/callbacks.rb:778:in `_run_call_callbacks'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/activesupport-4.2.9/lib/active_support/callbacks.rb:81:in `run_callbacks'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/actionpack-4.2.9/lib/action_dispatch/middleware/callbacks.rb:27:in `call'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/actionpack-4.2.9/lib/action_dispatch/middleware/remote_ip.rb:78:in `call'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/rollbar-2.14.1/lib/rollbar/middleware/rails/rollbar.rb:24:in `block in call'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/rollbar-2.14.1/lib/rollbar.rb:145:in `scoped'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/rollbar-2.14.1/lib/rollbar/middleware/rails/rollbar.rb:22:in `call'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/actionpack-4.2.9/lib/action_dispatch/middleware/debug_exceptions.rb:17:in `call'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/rollbar-2.14.1/lib/rollbar/middleware/rails/show_exceptions.rb:22:in `call_with_rollbar'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/actionpack-4.2.9/lib/action_dispatch/middleware/show_exceptions.rb:30:in `call'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/railties-4.2.9/lib/rails/rack/logger.rb:38:in `call_app'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/railties-4.2.9/lib/rails/rack/logger.rb:20:in `block in call'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/activesupport-4.2.9/lib/active_support/tagged_logging.rb:68:in `block in tagged'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/activesupport-4.2.9/lib/active_support/tagged_logging.rb:26:in `tagged'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/activesupport-4.2.9/lib/active_support/tagged_logging.rb:68:in `tagged'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/railties-4.2.9/lib/rails/rack/logger.rb:20:in `call'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/ahoy_matey-1.6.0/lib/ahoy/engine.rb:22:in `call_with_quiet_ahoy'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/quiet_assets-1.1.0/lib/quiet_assets.rb:27:in `call_with_quiet_assets'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/request_store-1.3.2/lib/request_store/middleware.rb:9:in `call'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/actionpack-4.2.9/lib/action_dispatch/middleware/request_id.rb:21:in `call'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/rack-1.6.8/lib/rack/methodoverride.rb:22:in `call'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/rack-1.6.8/lib/rack/runtime.rb:18:in `call'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/activesupport-4.2.9/lib/active_support/cache/strategy/local_cache_middleware.rb:28:in `call'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/rack-1.6.8/lib/rack/lock.rb:17:in `call'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/actionpack-4.2.9/lib/action_dispatch/middleware/static.rb:120:in `call'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/rack-1.6.8/lib/rack/sendfile.rb:113:in `call'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/railties-4.2.9/lib/rails/engine.rb:518:in `call'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/railties-4.2.9/lib/rails/application.rb:165:in `call'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/rack-1.6.8/lib/rack/urlmap.rb:66:in `block in call'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/rack-1.6.8/lib/rack/urlmap.rb:50:in `each'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/rack-1.6.8/lib/rack/urlmap.rb:50:in `call'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/rack-test-0.6.3/lib/rack/mock_session.rb:30:in `request'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/rack-test-0.6.3/lib/rack/test.rb:244:in `process_request'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/rack-test-0.6.3/lib/rack/test.rb:58:in `get'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/capybara-2.14.4/lib/capybara/rack_test/browser.rb:64:in `process'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/capybara-2.14.4/lib/capybara/rack_test/browser.rb:36:in `process_and_follow_redirects'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/capybara-2.14.4/lib/capybara/rack_test/browser.rb:22:in `visit'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/capybara-2.14.4/lib/capybara/rack_test/driver.rb:44:in `visit'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/capybara-2.14.4/lib/capybara/session.rb:269:in `visit'
     # /home/manuel/.rvm/gems/ruby-2.3.2/gems/capybara-2.14.4/lib/capybara/dsl.rb:50:in `block (2 levels) in <module:DSL>'
     # ./spec/features/polls/polls_spec.rb:238:in `block (3 levels) in <top (required)>'
     # ------------------
     # --- Caused by: ---
     # NameError:
     #   undefined local variable or method `with_hidden' for #<Poll::ActiveRecord_Relation:0x0055f8a558f4a0>
     #   /home/manuel/.rvm/gems/ruby-2.3.2/gems/activerecord-4.2.9/lib/active_record/relation/delegation.rb:136:in `method_missing'
```
which comes from comment model:
```ruby
  belongs_to :commentable, -> { with_hidden }, polymorphic: true, counter_cache: true
```